duckduckgo/macos-browser
# Fix passwords empty screen lock icon in dark mode

Task/Issue URL: https://app.asana.com/0/1203822806345703/1208570651540471/f

**Description**:

The lock icon on passwords screen empty state has wrong dark mode colour. 
This fixes it so it's the same colour as the text.

**Steps to test this PR**:
1. Make sure you have no passwords saved
2. Go to Password Management screen
3. Check that the icon matches the text colour.
4. Switch dark mode on and off to make sure this is the case in both states
5. Go to Settings -> Passwords + Autofill
6. Switch between Auto-lock on and off and make sure the empty-state message with the lock icon looks correct and that the link works.

**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
